pkg/features/feature_diff_summary.go

```diff
@@ -0,0 +1,102 @@
+package features
+
+import (
+	"sort"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+type ReleaseFeatureDiffInfo struct {
+	featureInfo map[string]map[string]*FeatureGateDiffInfo
+}
+
+func (i *ReleaseFeatureDiffInfo) AllFeatureInfo() []*FeatureGateDiffInfo {
+	ret := []*FeatureGateDiffInfo{}
+	for _, forClusterProfile := range i.featureInfo {
+		for featureSet := range forClusterProfile {
+			ret = append(ret, forClusterProfile[featureSet])
+		}
+	}
+	return ret
+}
+
+func (i *ReleaseFeatureDiffInfo) AllFeatureSets() sets.String {
+	allFeatureSets := sets.String{}
```

Comment:
This has been deprecated in favor of the generic version. There's a PR out to update library-go: https://github.com/openshift/library-go/pull/1643#pullrequestreview-1948942910